app/models/file_search.rb

```diff
@@ -19,6 +19,23 @@ def venue_names
     params.fetch(:venues) { [] }
   end
 
+  def self.year_range_limit
+    1935..Date.today.year
```

Comment:
We're getting `year_created` from the `date_created` property, which is a string.  In order to compute the `year_created` field, we're using `Date.parse` on the string, so using `Date.today` would be consistent with that, I believe.  I'll keep it as-is for now, but we might want to investigate further later.